client-common/src/main/java/org/apache/cassandra/sidecar/common/response/CompactionStopResponse.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.response;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.cassandra.sidecar.common.DataObjectBuilder;
+import org.apache.cassandra.sidecar.common.data.CompactionStopStatus;
+import org.apache.cassandra.sidecar.common.data.CompactionType;
+
+/**
+ * Response class for the Compaction Stop API
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CompactionStopResponse
+{
+    public static final String COMPACTION_TYPE_KEY = "compaction_type";
+    public static final String COMPACTION_ID_KEY = "compaction_id";
+    public static final String STATUS_KEY = "status";
+    private final CompactionType compactionType;
+    private final String compactionId;
+    private final CompactionStopStatus status;
+
+    private CompactionStopResponse(Builder builder)
+    {
+        this.compactionType = builder.compactionType;
+        this.compactionId = builder.compactionId;
+        this.status = builder.status;
+    }
+
+    /**
+     * Constructs a new {@link CompactionStopResponse}.
+     *
+     * @param compactionType the type of compaction that was requested to stop
+     * @param compactionId   the ID of the compaction that was requested to stop
+     * @param status         the status of the stop operation (e.g., "PENDING", "FAILED")
+     */
+    @JsonCreator
+    public CompactionStopResponse(@JsonProperty(COMPACTION_TYPE_KEY) CompactionType compactionType,
```

Comment:
Nit: can you reuse these in request payload too. Feel free to ignore this comment.